src/components/HeaderTabs/HeaderTabs.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Tab from './Tab';
+
+import HEADER_TABS from '../../constants';
+import { TabsWrapper } from './style';
+
+const HeaderTabs = ({ value, onClickTab }) => (
```

Comment:
I think we should rename this component to `HeaderNav`, and any link/button inside of this component will have `NavLink` name. WDYT?